tslint.json

```diff
@@ -0,0 +1,20 @@
+{
+  "extends": ["tslint:recommended", "tslint-react", "tslint-config-prettier"],
+  "rules": {
+    "curly": false,
+    "interface-name": [true, "never-prefix"],
+    "jsx-boolean-value": [true, "never"],
+    "jsx-no-lambda": false,
+    "jsx-no-bind": false,
+    "max-classes-per-file": [false],
+    "member-access": [false],
+    "member-ordering": false,
+    "no-unused-expression": false,
+    "no-var-requires": false,
+    "object-literal-sort-keys": false,
+    "ordered-imports": false,
+    "no-namespace": [true, "allow-declarations"],
+    "trailing-comma": false,
+    "variable-name": [true, "ban-keywords", "allow-leading-underscore"]
+  }
```

Comment:
Taken verbatim from reaction. We can can clean this up at some point.

Reply:
Yeah, there's a lot of config stuff shared across projects. Would be good to figure out a good way to share those.

Reply:
Totally agree, this is also dup'd in force now too